library/src/test/java/ch/decent/sdk/api/MessagingSuite.kt

```diff
@@ -81,4 +81,14 @@ class MessagingApiTest(channel: Channel) : BaseApiTest(channel) {
           .all { it.encrypted } `should equal` true
     }
   }
+
+  @Test fun `should get messages by id`() {
```

Comment:
I am not sure that order of tests is guaranteed, please add here also sending of expected messages.

Reply:
Please ignore after discussion